src/freadR.c

```diff
@@ -241,7 +241,7 @@ static void applyDrop(SEXP items, int8_t *type, int ncol, int dropSource) {
   UNPROTECT(1);
 }
 
-bool userOverride(int8_t *type, lenOff *colNames, const char *anchor, int ncol)
+bool userOverride(int8_t *type, lenOff *colNames, const char *anchor, const int ncol)
```

Comment:
there's a compiler warning for declaring const incorrectly right? what about variables that _could_ be const, is that something that could be detected?

Reply:
AFAIU more of the variables we are using should be constant as we are more commonly reading values and not writing.

Reply:
yep... any tool we can use to help us increase usage?

Reply:
I don't think it is that much important to hunt for such cases.